dev-tools/reconfigurator-cli/src/main.rs

```diff
@@ -1124,14 +1128,32 @@ fn cmd_load(
     for (sled_id, sled_details) in
         loaded.planning_input.all_sleds(SledFilter::Commissioned)
     {
-        if current_planning_input.sled_resources(&sled_id).is_some() {
-            swriteln!(
-                s,
-                "sled {}: skipped (one with \
-                the same id is already loaded)",
-                sled_id
-            );
-            continue;
+        match current_planning_input
+            .sled_lookup(SledFilter::Commissioned, sled_id)
+        {
+            Ok(_) => {
+                swriteln!(
+                    s,
+                    "sled {}: skipped (one with \
+                     the same id is already loaded)",
+                    sled_id
+                );
+                continue;
+            }
+            Err(error) => match error.kind() {
+                SledLookupErrorKind::Filtered { .. } => {
+                    swriteln!(
+                        s,
+                        "error: load sled {}: turning a decommissioned sled \
+                         into a commissioned one is not supported",
```

Comment:
Help me grok this a little -- this is a load command, right? How would this convert the status of a sled -- doesn't this command just print properties from a file?

Reply:
Ah what I meant here was within the scope of the reconfigurator sim, that for a particular sled:

* if the current planning input has it decommissioned
* but the loaded input has it commissioned

then we now print an error. Suggestions on a better way to phrase this?

Reply:
Chatted offline about this, I think this error is reasonable. I didn't previously see that the `cmd_load` function was modifying a simulated system.